src/amo/components/AddonReviewCard/index.js

```diff
@@ -545,6 +551,21 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
       >
         <div className="AddonReviewCard-container">
           {errorHandler.renderErrorIfPresent()}
+          {/* Div to display Addon's name and icon */}
+          {this.props.isUserProfile && addonName && addonIconURL && (
```

Comment:
add a condition for `this.isReply()` too - we don't want to show the addon name/icon for developer replies